htdocs/libraries/icms/core/Session.php

```diff
@@ -446,43 +448,32 @@ private function readSession($sess_id) {
 
 	/**
 	 * Inserts a session into the database
-	 * @param   string  $sess_id
-	 * @param   string  $sess_data
-	 * @return  bool
-	 **/
+	 *
+	 * @param string $sess_id
+	 * @param string $sess_data
+	 * @return bool
+	 */
 	private function writeSession($sess_id, $sess_data) {
 		$sess_id = icms::$xoopsDB->quoteString($sess_id);
 		$sess_data = icms::$xoopsDB->quoteString($sess_data);
 
-		$sql = sprintf(
-			"UPDATE %s SET sess_updated = '%u', sess_data = %s WHERE sess_id = %s",
-			icms::$xoopsDB->prefix('session'), time(), $sess_data, $sess_id
-			);
+		$sql = sprintf("UPDATE %s SET sess_updated = '%u', sess_data = %s WHERE sess_id = %s", icms::$xoopsDB->prefix('session'), time(), $sess_data, $sess_id);
 		icms::$xoopsDB->queryF($sql);
 		if (!icms::$xoopsDB->getAffectedRows()) {
-			$sql = sprintf(
-				"INSERT INTO %s (sess_id, sess_updated, sess_ip, sess_data)"
-				. " VALUES (%s, '%u', %s, %s)",
-				icms::$xoopsDB->prefix('session'),
-				$sess_id, time(),
-				icms::$xoopsDB->quoteString($_SERVER['REMOTE_ADDR']),
-				$sess_data
-			);
+			$sql = sprintf("INSERT INTO %s (sess_id, sess_updated, sess_ip, sess_data)" . " VALUES (%s, '%u', %s, %s)", icms::$xoopsDB->prefix('session'), $sess_id, time(), icms::$xoopsDB->quoteString($_SERVER['REMOTE_ADDR']), $sess_data);
```

Comment:
I would prefer https://www.php-fig.org/psr/psr-12/#23-lines PSR-12 or some similar rule to keep up with line length - it's easier to read.

Reply:
Yeah, I don't like how the  formatter changed these lines either.

Eclipse does a fairly decent job and have configured the rules from the [ImpressCMS code style](https://www.impresscms.org/modules/simplywiki/index.php?page=Style_Guide) . How it handles arrays now is almost impossible to get to work as I want it to. And, in this file, arrays are arguments for another function, so it splits them even more.